docs/components/concepts/process-instance-creation.md

```diff
@@ -114,6 +114,10 @@ By default, the instruction starts before the given element. This means input ma
 Multiple instructions can be provided to start the process instance at more than one element.
 You can activate the same element multiple times inside the created process instance by referring to the same element id in more than one instruction.
 
+:::note
+Start instructions have the same [limitations as the process instance modification](/components/concepts/process-instance-modification.md#limitations), e.g., it is not possible to start an element in a multi-instance subprocess.
```

Comment:
> e.g., it is not possible to start an element in a multi-instance subprocess.

This example may not be future-proof. There may be a time when we'll support this. Let's use `it is not possible to start at a sequence flow` instead, as we're unlikely to support that.